pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -186,6 +189,49 @@ func init() {
 	}
 
 	err = InfraSAs.addServiceAccount(
+		InfraImageTriggerControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: ImageTriggerControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// List Watch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams"),
+				},
+				// Spec update on triggerable resources
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{extensionsGroup},
+					Resources: sets.NewString("daemonsets"),
+				},
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{extensionsGroup},
```

Comment:
Should we switch to use the apps group if the present PR is going to land in 3.6?